.env.example

```diff
@@ -1,23 +0,0 @@
-NEXT_PUBLIC_MAINNET_NODE_URL=https://ergonode-api.ergohost.io
```

Comment:
Is it necessary, and is it ok, to remove this file? it appears to contain important information for the frontend to connect to the backend and to the blockchain.